### PR TITLE
fix lint and require issues causing 'bin/rake ci' to fail

### PR DESCRIPTION
**Pull request by thewoolleyman**
This change is [Reviewable](https://reviewable.io/reviews/shakacode/react-webpack-rails-tutorial/268)

### CODE REVIEW

**Comment by thewoolleyman** [comment]
I don't know why these didn't fail on travis CI, but they failed for me locally.

File: client/app/bundles/comments/components/CommentBox/CommentList/CommentList.spec.jsx [hunk 1]
```diff
@@ -28,7 +28,7 @@ describe('CommentList', () => {
       id: 2,
       author: 'Furter',
       text: 'ho',
-    }),
+    })
```
Comment (justin808) [inline client/app/bundles/comments/components/CommentBox/CommentList/CommentList.spec.jsx]: I think we're supposed to have a trailing comma.

File: spec/rails_helper.rb [hunk 2]
```diff
@@ -7,6 +7,7 @@
 require "rspec/rails"
 require "capybara/rspec"
 require "capybara-screenshot/rspec"
+require "database_cleaner"
```
Comment (justin808) [inline spec/rails_helper.rb]: this is odd -- are we sure this is needed?

File: client/app/bundles/comments/components/CommentBox/CommentList/CommentList.spec.jsx (hunk 1 above)
Comment (thewoolleyman) [inline client/app/bundles/comments/components/CommentBox/CommentList/CommentList.spec.jsx]: The linter failed on the trailing comma when I ran locally, with latest code, with just a bundle and npm install and rake ci.  If it doesn't for you or on travis, then there's something non-deterministic that is bleeding in from those environments or mine, and that should be locked down / prevented.  I didn't dig into why, I just made this change and the linter passed.

File: spec/rails_helper.rb (hunk 2 above)
Comment (thewoolleyman) [inline spec/rails_helper.rb]: Same as above, it blew up for me on a clean rake ci of master until I added this.  If it didn't on travis or for you, there's something non-deterministic.

File: client/app/bundles/comments/components/CommentBox/CommentList/CommentList.spec.jsx (hunk 1 above)
Comment (justin808) [inline client/app/bundles/comments/components/CommentBox/CommentList/CommentList.spec.jsx]: @thewoolleyman in some cases, we tried using docker for linting. I don't like that it as it doesn't seem to lock down the linter versions.

File: spec/rails_helper.rb (hunk 2 above)
Comment (justin808) [inline spec/rails_helper.rb]: I'll wait to see if other's report this.

File: client/app/bundles/comments/components/CommentBox/CommentList/CommentList.spec.jsx (hunk 1 above)
Comment (thewoolleyman) [inline client/app/bundles/comments/components/CommentBox/CommentList/CommentList.spec.jsx]: OK, this one isnlt failing any more against master, but the other one is.  I've removed this change.

File: spec/rails_helper.rb (hunk 2 above)
Comment (thewoolleyman) [inline spec/rails_helper.rb]: This is still a problem for me against latest master.  I don't know what else is requiring database cleaner without this line.  If it works for you, I'd be interested in seeing what requires it if you put a raise at the top of the database cleaner gem entry point file and post the stacktrace.

**Comment by justin808** [comment]
:lgtm:

---

Reviewed 1 of 2 files at r1, 1 of 1 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/shakacode/react-webpack-rails-tutorial/268#-:-KFmZvJgXTcOua8Nv6sZ:1417088721)*

**Comment by justin808** [comment]
@thewoolleyman can you rebase this on top of master?

**Comment by thewoolleyman** [comment]
@justin808 Rebased and pushed...

**Comment by justin808** [comment]
:lgtm:

---

Reviewed 1 of 1 files at r3.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/shakacode/react-webpack-rails-tutorial/268#-:-KFoheYYG7aoBlCZTuDs:1417088721)*

**Comment by justin808** [comment]
Thanks @thewoolleyman 

---

Review status: :shipit: all files reviewed at latest revision, all discussions resolved, all commit checks successful.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/shakacode/react-webpack-rails-tutorial/268#-:-KFohqnoJVlcjPlCWJp7:-2106647794)*

**Pull request merged**